crates/ruff_linter/src/rules/pylint/rules/eq_without_hash.rs

```diff
@@ -11,12 +11,10 @@ use crate::checkers::ast::Checker;
 ///
 /// ## Why is this bad?
 /// A class that implements `__eq__` but not `__hash__` will have its hash
-/// method implicitly set to `None`. This will cause the class to be
-/// unhashable, will in turn cause issues when using the class as a key in a
-/// dictionary or a member of a set.
-///
-/// ## Known problems
-/// Does not check for `__hash__` implementations in superclasses.
+/// method implicitly set to `None`, regardless of if a super class defines
+/// `__hash__`. This will cause the class to be unhashable, will in turn
+/// cause issues when using the class as a key in a dictionary or a member
+/// of a set.
```

Comment:
I think we should provide an additional example below for the case where a superclass has defined a `__hash__` method and still the subclass is unhashable because Python will not inherit it and provide a link to https://docs.python.org/3/reference/datamodel.html#object.__hash__.